Makefile

```
@@ -136,3 +138,24 @@ verify: check check-docforge format test
 
 .PHONY: verify-extended
 verify-extended: check-generate check check-docforge format test test-cov test-clean
+
+# skaffold dev and debug clean up deployed modules by default, disable this
+debug: export SKAFFOLD_CLEANUP = false
+# Artifacts might be already built when you decide to start debugging.
+# However, these artifacts do not include the gcflags which `skaffold debug` sets automatically, so delve would not work.
+# Disabling the skaffold cache for debugging ensures that you run artifacts with gcflags required for debugging.
+debug: export SKAFFOLD_CACHE_ARTIFACTS = false
+
+debug: export SOURCE_DATE_EPOCH = $(shell date -d $(BUILD_DATE) +%s)
+
+.PHONY: debug
+debug: $(SKAFFOLD)
+	@echo "LD_FLAGS: $(LD_FLAGS_DEBUG)"
+	@LD_FLAGS=$(LD_FLAGS) $(SKAFFOLD) debug
```

Cosmetic nit: If you want to see the passed var, you could use

```suggestion
	LD_FLAGS=$(LD_FLAGS) $(SKAFFOLD) debug
```

The `@` prefix silences printing the command that gets executed.

---

Thanks for the pointer. The printout was strictly a temporary debug aid for initial project setup, so I'm removing it.